tools/abel/README.md

```diff
@@ -123,6 +134,12 @@ file. Only required for the `Building Config -> Spreadsheet` workflow.
   may be provided. Only required if the token is stored in a directory that isn't
   the current directory.
 
+`-p` or `--subscription` fully-qualified path to a Google Cloud Pubsub subscription, e.g. `projects/google.com:your-project/subscriptions/your-subscription`. This parameter is optional and it only required to run telemetry validation in a newly generated building config.
```

Comment:
this could be removed?

Reply:
This is needed for telemetry validation if the user isn't using application-default authentication, no?

https://github.com/google/digitalbuildings/tree/master/tools/validators/instance_validator#telemetry-validation

Reply:
ok can you remove `is optional and it is only required` to this parameter is only required for telemetry validation.
it does not need to be newly generated right?

Reply:
Done.

Reply:
I dont see it updated, did you push it?